.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: objective-c  #1
+osx_image: xcode7.3 #2
+script: xctool -project Choose-Lunch.xcodeproj -scheme Choose-Lunch build  CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
```

Comment:
Added as Travis tried to code sign the project, this disables and build's via Xcode's tools.